reports.js

```diff
@@ -0,0 +1,62 @@
+const axios = require('axios');
+const pulls = require('./pulls.js');
+const updatesUrl = 'http://updates.jenkins.io/plugin-documentation-urls.json';
+const installsUrl = 'https://stats.jenkins.io/jenkins-stats/svg/201910-plugins.csv';
+const httpCache = {};
+
+/**
+ * Get the content of the progress report
+ * @return {array} of objects representing table rows
+ */
+async function pluginsReport() {
+  const documentation = await getContent(updatesUrl, 'json');
+  const installs = await getContent(installsUrl, 'blob');
+  const lines = installs.split('\n');
+  lines.forEach(function(line) {
+    const nameAndValue = JSON.parse('[' + line + ']');
+    const plugin = documentation[nameAndValue[0]] || {};
+    plugin.installs = nameAndValue[1];
+  });
+
+  const report = [];
+  Object.keys(documentation).forEach(function(key) {
+    const plugin = documentation[key];
+    const url = documentation[key].url || '';
+    plugin.name = key;
+    if (url.match('https?://github.com/jenkinsci/')) {
+      plugin.status = 'OK';
+      plugin.className = 'success';
+    } else if (pulls[key]) {
```

Comment:
I'm not sure I quite understand the point of this hash, its just a list of hard coded PRs that have already been merged? i guess pending a new release?

Reply:
The idea is to only have 3 states: TODO, PR (not merged or merged but unreleased), OK. Hopefully it's useful to avoid more people working on the same thing.  The downside is that the PR hash needs to be updated in code.